backend/tests/test_users_router.py

```diff
@@ -131,9 +131,7 @@ def test_get_by_id(self, clean_db):
                 f"INSERT INTO users (name, email, password, address, phone) VALUES ({data_string}) RETURNING id;"

```

Comment:
why is it raw sql here and not ORM-style query?

Reply:
Test was removed to be allow migration and prevent SW rot.
Full test-suite will be added in a different branch.